pkg/util/scrubber/scrubber.go

```diff
@@ -40,6 +42,17 @@ type Replacer struct {
 	// ReplFunc, if set, is called with the matched bytes (see regexp#Regexp.ReplaceAllFunc). Only
 	// one of Repl and ReplFunc should be set.
 	ReplFunc func(b []byte) []byte
+
+	// ImplementVersion is the version of the implementation of the replacer.
+	ImplementVersion *version.Version
+}
+
+func parseVersion(versionString string) *version.Version {
+	v, err := version.New(versionString, "")
+	if err != nil {
+		return nil
```

Comment:
What about `panic`king here? Since this is used to parse a compile-time variable, I think this would need to panic if the value to parse is incorrect.

Reply:
Good idea